src/gui/styles/style_constants.rs

```diff
@@ -177,23 +176,26 @@ pub const SARASA_MONO: Font = Font {
 };
 
 pub fn get_font(style: StyleType) -> Font {
-    match to_rgb_color(get_colors(style).text_body) {
-        RGBColor(255, 255, 255) => SARASA_MONO,
-        _ => SARASA_MONO_BOLD,
+    if style.is_nightly() {
+        SARASA_MONO
+    } else {
+        SARASA_MONO_BOLD
     }
 }
 
 pub fn get_font_headers(style: StyleType) -> Font {
-    match to_rgb_color(get_colors(style).text_headers) {
-        RGBColor(255, 255, 255) => SARASA_MONO,
-        _ => SARASA_MONO_BOLD,
+    if style.is_nightly() {
+        SARASA_MONO_BOLD
+    } else {
+        SARASA_MONO
     }
 }
 
 pub fn get_color_mix_chart(style: StyleType) -> f64 {
```

Comment:
Do you think I can remove `color_mixing` from `PaletteExtension`?

It seems like Night styles will have a mixing value of 0.3 and Day styles will have 0.8.

If that's the case, then `PaletteExtension` can be changed from:

```rust
pub struct PaletteExtension {
    /// Color of favorites star
    pub starred: Color,
    /// Badge/logo alpha
    pub badge_alpha: f32,
    /// Traffic chart color mixing
    pub color_mixing: f64,
}
```
To:
```rust
pub struct PaletteExtension {
    /// Color of favorites star
    pub starred: Color,
    /// Badge/logo alpha
    pub badge_alpha: f32,
}
```

And color mix to:
```rust
pub const fn get_color_mix_chart(style: StyleType) -> f64 {
    if style.is_nightly() {
        0.3
    } else {
        0.8
    }
}
```

Reply:
Actually it seems so... I wasn't thinking about it and I chose similar values when adjusting the extra palettes.
Anyway, I've merged that parameter with badge_alpha since they are practically the same thing.
I'm not sure we can generalise this to all night/day themes since it also depends on the contrast between the specific primary and secondary.
If proceeding we see that every night/day theme is around that value we'll remove it.